### PR TITLE
Update changelog for 2.2.0

### LINKED ISSUE #551: Update changelog for 2.2.0 beta 1

### PR DESCRIPTION
**Pull request by ocean90**
Changes: https://github.com/GlotPress/GlotPress-WP/milestone/9?closed=1

Fixes #551.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,33 @@
...
+
+**Features**
+
+* Add locale information for the Latin version of Hausa. ([#549](https://github.com/GlotPress/GlotPress-WP/pull/549))
+* Fix translations which are using the placeholder for tab characters. ([#473](https://github.com/GlotPress/GlotPress-WP/pull/473))
+* Add `gp_reference_source_url` filter for the source URL of a project. ([#522](https://github.com/GlotPress/GlotPress-WP/pull/522))
+* Provide minified assets. ([#505](https://github.com/GlotPress/GlotPress-WP/issue/505))
+* Update JavaScript library for table sorting. ([#502](https://github.com/GlotPress/GlotPress-WP/issue/502))
+
+Thanks to all the contributors so far: Alexander Kirk, David Decker, Dominik Schilling, Greg Ross,  Pedro Mendonça, Petya Raykovska, and Sergey Biryukov.
```
Comment (swissspidy) [inline CHANGELOG.md]: Nitpicking: Duplicated space before _Pedro_

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,33 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## 2.2.0-beta.1 (September 19, 2016)
+
+**Breaking Changes**
+
+* Change the slug of the Kyrgyz locale to `kir`. ([#550](https://github.com/GlotPress/GlotPress-WP/pull/550))
```
Comment (toolstack) [inline CHANGELOG.md]: Should we mention what the old slug was?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,33 @@
...
+**Bugfixes**
+
+* Fix broken cancel link on project create form. ([#547](https://github.com/GlotPress/GlotPress-WP/issues/547))
+* Fix native name of the Tibetan locale name from བོད་སྐད to བོད་ཡིག. ([#539](https://github.com/GlotPress/GlotPress-WP/pull/539))
+* Fix extra entry in `GP_Translation::translations()`. ([#516](https://github.com/GlotPress/GlotPress-WP/issues/516))
+* Merge similar strings to improve translation. [#535](https://github.com/GlotPress/GlotPress-WP/issues/535))
+* Refactor script and style registration to make them more reliable. [#476](https://github.com/GlotPress/GlotPress-WP/issues/476))
+* Update locale information for Kyrgyz to use correct data. ([#550](https://github.com/GlotPress/GlotPress-WP/pull/550))
+
+
```
Comment (toolstack) [inline CHANGELOG.md]: Extra blank line?

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,6 +1,33 @@
...
+**Breaking Changes**
+
+* Change the slug of the Kyrgyz locale to `kir`. ([#550](https://github.com/GlotPress/GlotPress-WP/pull/550))
+
+**Bugfixes**
+
+* Fix broken cancel link on project create form. ([#547](https://github.com/GlotPress/GlotPress-WP/issues/547))
+* Fix native name of the Tibetan locale name from བོད་སྐད to བོད་ཡིག. ([#539](https://github.com/GlotPress/GlotPress-WP/pull/539))
+* Fix extra entry in `GP_Translation::translations()`. ([#516](https://github.com/GlotPress/GlotPress-WP/issues/516))
+* Merge similar strings to improve translation. [#535](https://github.com/GlotPress/GlotPress-WP/issues/535))
```
Comment (toolstack) [inline CHANGELOG.md]: Missing bracket at the beginning of the PR#.

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,6 +1,33 @@
...
+* Refactor script and style registration to make them more reliable. [#476](https://github.com/GlotPress/GlotPress-WP/issues/476))
```
Comment (toolstack) [inline CHANGELOG.md]: Missing bracket at the beginning of the PR#.

File: CHANGELOG.md (hunk 1 above)
Comment (toolstack) [reply CHANGELOG.md]: Also, if we're using 'and' we can drop the final comma (in ones below as well).

Comment (ocean90) [reply CHANGELOG.md]: @toolstack That's the Oxford comma. 😉

Comment (toolstack) [reply CHANGELOG.md]: I can give you the name of an old English teacher I had that can go on for hours about why the comma is wrong before the 'and' :wink:

**Review by toolstack — APPROVED** [review]
:+1:

**Pull request merged**